packages/apps/docs/src/hooks/useWindowScroll.ts

```diff
@@ -11,7 +11,7 @@ export function useWindowScroll(): [
     y: 0,
   });
 
-  React.useLayoutEffect((): (() => void) => {
```

Comment:
useLayoutEffect can have sideeffects with SSR
only use for components that are only rendered on clientside